src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -224,9 +224,6 @@ func runClientProxy(cfg *config.Config) {
 	}
 
 	// Start user interface.
-	if cfg.UIAddr == "" {
```

Comment:
Remove extra check. Even if `cfg.UIAddr` is empty, next statement will also catch it.